.github/workflows/test-deploy.yaml

```diff
@@ -53,6 +53,12 @@ jobs:
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: eu-central-1
+    - name: Increment version
+      if: "!startsWith(github.event.head_commit.message, '[RELEASE]')"
+      steps:
+        - run: npm version patch -m "[RELEASE] %s"
```

Comment:
better to use yarn for incrementing versions, since it's already used in the project

Reply:
+++